resource/resource_manager.go

```diff
@@ -204,7 +204,6 @@ func (r ResourceManager) StopFileSyncService() {
 			case <-stopChan:
 				glog.V(5).Infof(rmLogString(fmt.Sprintf("Embedded ESS Stop completed.")))
 				done = true
-				return
```

Comment:
Question: why remove `return`?

Reply:
Because it was preventing the code below from running, which does additional cleanup of downloaded objects.